rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -120,4 +124,3 @@ install -D -m0640 %{buildroot}%{gem_instdir}/config/foreman_maintain.yml.packagi
 
 * Mon Feb 27 2017 Anurag Patel <apatel@redhat.com> 0.0.1-1
 - Package foreman_maintain into RPM (#3, apatel@redhat.com)
-
```

Comment:
This line is added back by tito when we tag it so could you keep it in to reduce the amount of changes?

Reply:
Ah! I had configured emacs to delete-trailing-white-space. Fixed now.

Reply:
Totally makes sense and normally I'd do exactly the same. For now this is what tito does though :(